ibis/backends/flink/datatypes.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import annotations
+
+import pyflink.table.types as fl
```

Comment:
```suggestion
from pyflink.table.types import DataTypes
```
Nit: Not a fan of just making up aliases unnecessarily, when they're not really used idiomatically anywhere. If you really want to namespace it (not necessary here, I think), at least `from pyflink.table import types` makes more sense to me.

Reply:
I think one of the benefits is to allow readers to easily distinguish between Ibis provided type primitives vs backend provided primitives to potentially avoid human mistakes
e.g., before code is clearer to me with the namespace, otherwise i have to guess what DataTypes.STRING() is.
```
if typ == fl.DataTypes.STRING():
       return dt.String(nullable=nullable)
```

Also [big query backend](https://github.com/ibis-project/ibis/blob/master/ibis/backends/bigquery/datatypes.py#L3) seems to use this convention as well. I don't have a strong opinion on this one. Can be useful to get a third opinion.

Reply:
Fair enough, I'll change to the recommendation.